backend/molgenis-emx2/src/main/java/org/molgenis/emx2/utils/TypeUtils.java

```diff
@@ -106,8 +106,10 @@ else if (v instanceof String) {
   public static Boolean toBool(Object v) {
     if (v == null) return null; // NOSONAR
     if (v instanceof String) {
-      if ("true".equalsIgnoreCase((String) v)) return true;
-      if ("false".equalsIgnoreCase((String) v)) return false;
+      if ("true".equalsIgnoreCase(((String) v).trim())
```

Comment:
unrelated feature/bugfix: we noticed often people saying 'yes' instead of 'true'